source/material_model/perplex_simpler.cc

```diff
@@ -0,0 +1,225 @@
+/*
+  Copyright (C) 2011 - 2017 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+#include <aspect/material_model/perplex_simpler.h>
+
+extern "C" { 
+#include <perplex_c.h>
```

Comment:
what if we don't find the perplex library? We have to guard the implementation of this class if we don't. We basically have to export ASPECT_WITH_PERPLEX or something (similar to ASPECT_USE_PETSC).